CHANGELOG.md

```diff
@@ -97,6 +97,8 @@ constructor. ([#6240](https://github.com/biocore/scikit-bio/issues/624))
 * ``Sequence`` objects now handle slicing of empty positional metadata correctly. Any metadata that is empty will no longer be propagated by the internal ``_to`` constructor. ([#1133](https://github.com/biocore/scikit-bio/issues/1133))
 * ``DissimilarityMatrix.plot()`` no longer leaves a white border around the
   heatmap it plots (PR #1070).
+* TreeNode.root_at_midpoint`` no longer fails when a node with two equal length child branches exists in the tree. ([#1077](https://github.com/biocore/scikit-bio/issues/1077))
+* ``TreeNode._set_max_distance`` would store distance information as ``list``s in the attribute ``MaxDistTips`` on each node, however, these distances were only valid for the node in which the call to ``_set_max_distance`` was called on. ([#1223](https://github.com/biocore/scikit-bio/issues/1223))
```

Comment:
No public methods were impacted by this, as in, I'm not aware of a method in scikit-bio that produced incorrect results due to #1223; no method that I'm aware of traversed the tree and interrogated `MaxDistTips`. A public attribute was impacted, and its impact is described.